packages/components-js/tests/vrt/playwright/helpers/cdp-helper.ts

```diff
@@ -99,16 +95,18 @@ const getHostElementNodeIds = async (cdp: CDPSession, selector: string): Promise
   ).nodeIds;
 };
 
-const findBackendNodeIds = (currentNode: Protocol.DOM.Node, localNodeNameOrClassName: string): BackendNodeId[] => {
-  // support tag names & class names
+const findBackendNodeIds = (currentNode: Protocol.DOM.Node, selector: string): BackendNodeId[] => {
+  const attributes = currentNode.attributes || [];
+  // support tag names, ids & class names
   if (
-    currentNode.localName === localNodeNameOrClassName ||
-    currentNode.attributes?.includes(localNodeNameOrClassName)
+    currentNode.localName === selector ||
+    (selector.includes('#') && selector.replace('#', '') === attributes[attributes.indexOf('id') + 1]) ||
```

Comment:
Is includes a good choice? Can the `#` or `.` be located somewhere in the middle or does it have to start like this?